worker/migrations/migrations.go

```diff
@@ -80,6 +91,137 @@ func commit(ctx *job.WorkerContext, err error) error {
 	return err
 }
 
+// Migrate all the encrypted accounts to Bitwarden ciphers.
+// It decrypts each account, reencrypt the fields with the organization key,
+// and save it in the ciphers database.
+func migrateAccountsToOrganization(domain string) error {
+	inst, err := instance.GetFromCouch(domain)
+	if err != nil {
+		return err
+	}
+	setting, err := settings.Get(inst)
+	if err != nil {
+		return err
+	}
+	// Get org key
+	var orgKey []byte
+	orgKey, err = setting.OrganizationKey()
+	if err == settings.ErrMissingOrgKey {
+		// Create org key if missing
+		if err := setting.EnsureCozyOrganization(inst); err != nil {
+			return err
+		}
+		orgKey, err = setting.OrganizationKey()
+		if err != nil {
+			return err
+		}
+	} else if err != nil {
+		return err
+	}
```

Comment:
In fact, I think we can simplify this code to:

```go
if err := setting.EnsureCozyOrganization(inst); err != nil {
    return err
}
orgkey, err := setting.OrganizationKey()
if err != nil {
    return err
}
```

Reply:
In general, `EnsureSomething` methods can be called safely (they are idempotent).